index.js

```diff
@@ -25,4 +25,137 @@ mobileMenuLinks.forEach((link) => {
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+  },
+];
+
+for (let i = 0; i < projectInfos.length; i += 1) {
+  const projectCard = document.createElement('div');
+  projectCard.classList.add('project-card');
+  let technologiesList = '';
+  for (let j = 0; j < projectInfos[i].technologies.length; j += 1) {
+    technologiesList += `<li>${projectInfos[i].technologies[j]}</li>`;
+  }
+
+  projectCard.innerHTML = `
+      <div class="project-card">
+        <div class="project-card_image">
+          
+        </div>
+        <div class="project-card_details">
+          <h3>${projectInfos[i].title}</h3>
+          <div class="stacks">
+            <ul>
+              ${technologiesList}
+            </ul>
+          </div>
+          <button class="green-button">See project</button>
+        </div>
+      </div>
+  `;
+
+  card.appendChild(projectCard);
+}
+
+const poppedUpCard = document.querySelector('.popup-card');
+const popUpButton = document.getElementsByClassName('green-button');
+const popUpCard = document.createElement('div');
+
+for (let i = 0; i < popUpButton.length; i += 1) {
+  popUpButton[i].addEventListener('click', () => {
+    poppedUpCard.style.display = 'block';
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+
+    let technologies = '';
+    for (let j = 0; j < projectInfos[i].technologies.length; j += 1) {
+      technologies += `<li>${projectInfos[i].technologies[j]}</li>`;
+    }
+
+    popUpCard.innerHTML = `
+        <div class="modal-body">
+          <img src="./images/Icon.svg" class='close-btn'>
+          <img class="modal-image" src="${projectInfos[i].image}" />       
+          <div class="modal-desktop">
+            <div class="title-button">
+              <h2 class="modal-header">${projectInfos[i].title}</h2>
+              <div class="modal-buttons">
+                <button class="modal-button">See live <img src="./images/live-icon.svg"></button>
+                <button class="modal-button">See source <img src="./images/github-icon.svg"></button>
+              </div>
+              </div>
+          </div>
+          <ul class="technologies-list">
+            ${technologies}
+          </ul>
+          <div class="card-p">
+            <p>${projectInfos[i].description}</p>
+          </div>
+          <div class="mobile-buttons">
+            <button class="mobile-button">See live <img src="./images/live-icon.svg"></button>
+            <button class="mobile-button">See source <img src="./images/github-icon.svg"></button>
+          </div>
```

Comment:
- [ ] You did fantastic work generating the contents dynamically using javascript. :+1:  It would be better to use links instead of buttons as we are trying to provide the live version and the source of our projects. Kindly ensure that you have attributes named `link` and `source` in your objects, that way you can use them as the `href` attribute for the `a` tags. 🤝️